core-data/selectors.js

```diff
@@ -66,7 +71,21 @@ export function getMedia( state, id ) {
  * @return {Array} Authors list.
  */
 export function getAuthors( state ) {
-	return state.authors;
+	return getUserQueryResults( state, 'authors' );
+}
+
+/**
+ * Returns all the users returned by a query ID.
+ *
+ * @param {Object} state   Data state.
+ * @param {string} queryID Query ID.
+ *
+ * @return {Array} Users list.
+ */
+export function getUserQueryResults( state, queryID ) {
```

Comment:
This selector should probably be memoized to avoid generating an new array on each call.